Tests/AppTests/__Snapshots__/GithubTests/test_fetchReadme_withPrivateGitHubImages.1.txt

```diff
@@ -0,0 +1,6 @@
+<html>
+ <head></head>
+ <body> 
+  <img src="https://awsReadmeBucket.s3.us-east-2.amazonaws.com/foo/bar/with-jwt.jpg" data-original-src="https://private-user-images.githubusercontent.com/with-jwt.jpg?jwt=some-jwt" /> 
+ </body>
+</html>
```

Comment:
It looks like this was a snapshot for a test in `GithubTests` in an earlier version but isn't anymore? There's no `GithubTests.test_fetchReadme_withPrivateGitHubImages`.

Reply:
Well spotted. Thanks